test/test_models.py

```diff
@@ -97,3 +98,19 @@ def test_add_stage_too_far_ahead(sample_with_stages):
     (stages, token) = models.get_sample_stages(s.obfuscated_id)
     assert 2 == len(stages)
     assert h.encode(2) == token
+
+
+def test_create_first_stage_file(storepath, sample_with_stages):
+    stage = sample_with_stages['stages'][0]
+    ssf = models.SampleStageFile(stage)
+    assert '00001/00001/00001-00000' == ssf.relative_file_path
```

Comment:
Just so I'm clear, the structure tested here is [project ID]/[sample ID]/[stage ID]-[file ID], correct? If so, I'm good with that.